e2e/integration/devconsoletests.js

```diff
@@ -0,0 +1,28 @@
+context( 'Test devloper console', () => {
+    before( () => {
+        cy.visit( Cypress.env( 'QA_HOST' ) + 'console');
+        cy.get( '#username', { timeout: 5000 } ).type( Cypress.env( 'USERNAME' ) );
+        cy.get( '#password' ).type( Cypress.env( 'PASSWORD' ) );
+        cy.get( '#submit' ).click();
+    } );
+    Cypress.on( 'uncaught:exception', ( err, runnable ) => {
+        return false;
+    } );
+
+    it( 'Test for create the application', () => {
+        cy.contains( 'Create new App', { timeout: 5000 } ).click()
+        cy.get( '#app-name' ).type( 'e2e test automation' )
+        cy.get( '#app-path' ).type( 'e2e test automation' )
+        cy.get( '#app-desc' ).type( 'e2e test automation' )
+        cy.contains( 'Create App' ).click()
+        cy.contains( 'App Created Succssfully!' ).should( 'be.visible' );
```

Comment:
```suggestion
        cy.contains( 'App Created Successfully!' ).should( 'be.visible' );
```

Reply:
@riginoommen We need to spell it correctly in the application pop-up, I used the same for assertion!

Reply:
@supriyagawade Let's put the correct word in the test cases. we can ask @deshmukhmayur to update the message.